app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RegistrationsController < Devise::RegistrationsController
+  private
+
+  def sign_up_params
+    params.require(:user).permit(:name, :username, :email, :password, :password_confirmation)
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)